<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>381</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="genTab">
    <attribute name="title">
     <string>Zoom Shortcut Generator</string>
    </attribute>
    <widget class="QPushButton" name="submitButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>160</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="urlEnt">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameEnt">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="urlLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;URL&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="modeLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Mode&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="nameLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="modeInput">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <item>
      <property name="text">
       <string>Create</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Delete</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="exTab">
    <attribute name="title">
     <string>Zoom Link Extractor</string>
    </attribute>
    <widget class="QLabel" name="nameLabel2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameEnt2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="submitButton2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>120</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Extract Link</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="helpButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
